spec/features/changing_answers_spec.rb

```diff
@@ -242,4 +242,40 @@
     expect(claim.student_loan_start_date).to eq StudentLoans::BEFORE_1_SEPT_2012
     expect(claim.student_loan_plan).to eq StudentLoans::PLAN_1
   end
+
+  context "with a field that can be verified" do
```

Comment:
More general points about specs and nested contexts: ideally, nesting of `describe` and `context` should be kept to a minimum as deeply nested specs can be difficult to maintain, read and follow. I think we should avoid going more than a single level deep where possible. In these cases below, we only have one actual scenario per two nested contexts, and so I'd suggest they could be re-written much more simply without any contexts at all, e.g.

`scenario "user cannot change the value of an identity field that was acquired from Verify"` 

and

`scenario "user can change the answer to an identity question that wasn't acquired from Verify"`